privateer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "servers": [
+        {
+            "name": "annex2",
+            "hostname": "annex2.montagu.dide.ic.ac.uk",
+            "port": 10023,
```

Comment:
just to double check this port is correct right? usually use 10022 but you may have it exposed on 10023

Reply:
can be any port at all - nothing special about ports this high